src/test/java/org/broadinstitute/hellbender/tools/walkers/annotator/BaseQualityRankSumTestUnitTest.java

```diff
@@ -46,8 +47,17 @@ private GATKRead makeRead(final byte qual) {
         return read;
     }
 
-    @Test
+    @DataProvider(name = "data")
+    private Object[][] data(){
+        return new Object[][]{
+                {},
```

Comment:
What the heck is this? :) Why even have a `DataProvider` if your only data is an empty array?